meta-mender-core/recipes-bsp/systemd-mender-config/files/ab_setup.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
```

Comment:
The license checker is complaining that this file doesn't have a copyright header. The easiest, if it's ok with you, is simply to assign copyright to us, and add [this header](https://github.com/mendersoftware/meta-mender/blob/master/tests/acceptance/test_build.py#L2-L14). It makes little practical difference, since this is anyway Apache licensed.

If you would like to keep the copyright, we can also arrange that, but it will take me a day or two to fix our license checker (being an Open Source project, it's something it should cope with).

Reply:
After getting approval from the appropriate lawyers, I have added the requisite license information.